sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/V1Beta3.java

```diff
@@ -531,11 +531,17 @@ private RunQueryRequest makeRequest(Query query) {
 
     /**
      * Datastore system tables with statistics are periodically updated. This method fetches
-     * the latest timestamp of statistics update using the {@code __Stat_Total__} table.
+     * the latest timestamp of statistics update using the {@code __Stat_Total__} or
+     * {@code __Stat_Ns_Total__} table.
      */
     private long queryLatestStatisticsTimestamp(Datastore datastore) throws DatastoreException {
       Query.Builder query = Query.newBuilder();
-      query.addKindBuilder().setName("__Stat_Total__");
```

Comment:
I think it should work, but I want to verify it with an actual project that exposes these stats. Will create a new project and give it a try.